src/Utilities/Spherepack.hpp

```diff
@@ -0,0 +1,112 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+/// This is a C++ interface to SPHEREPACK, which is written in FORTRAN.
```

Comment:
Rather than a standalone Doxygen comment, I would suggest this either be:
* part of the comment block below, or
* a `\file` Doxygen comment, in which case I would use the wording `Provides a C++ interface to the Fortran SPHEREPACK library`.